java/src/org/openqa/selenium/grid/node/docker/DockerOptions.java

```diff
@@ -181,7 +181,8 @@ public Map<Capabilities, Collection<SessionFactory>> getDockerSessionFactories(
                     videoImage,
                     assetsPath,
                     networkName,
-                    info.isPresent()));
+                    info.isPresent(),
+                    options.getSlotMatcher()));
```

Comment:
We should pass a `Predicate` here, in the same way it was done at the `LocalNodeFactory`.

Reply:
updated